Kudu.Contracts/SiteExtensions/ISiteExtensionManager.cs

```diff
@@ -17,7 +17,7 @@ public interface ISiteExtensionManager
         /// <summary>
         /// Install or update a site extension
         /// </summary>
-        Task<SiteExtensionInfo> InstallExtension(string id, string version, string feedUrl, SiteExtensionInfo.SiteExtensionType type, ITracer tracer);
+        Task<SiteExtensionInfo> InstallExtension(string id, string version, string feedUrl, SiteExtensionInfo.SiteExtensionType type, ITracer tracer, string installationArgs);
```

Comment:
Is there any versioning requirements on updating the public interface?
I'd suggest to add comment clarifying whether it's a one string argument or it will be parsed to multiple arguments. In other words for `installationArgs = "a b"` will the input for `install.cmd` be `%1 = a` and `%2 = b` or `%1 = a b`?